MB-System build fails due to missing GMT header file gmt_remote.h

I'm following up on a bug report to me - a user is trying to build a new MB-System pre-release distribution with GMT downloaded from your master branch (which is to say, bleeding edge stacked on bleeding edge...). 

I've verified this using the state of the GMT master branch a bit earlier this afternoon. The downloaded current GMT was build and installed without issue. The MB build fails fails with the error message:

libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../src/mbio -I../../src/mbio -I../../src/mbtrn/r7kr -I../../src/mbtrn/utils -I../../src/mbtrn/mframe/src -I/usr/local/include/gmt -I/usr/local/Cellar/gdal/2.4.4_4/include -I/usr/local/Cellar/netcdf/4.7.4/include -g -I/opt/X11/include -MT mb_readwritegrd.lo -MD -MP -MF .deps/mb_readwritegrd.Tpo -c mb_readwritegrd.c  -fno-common -DPIC -o .libs/mb_readwritegrd.o
In file included from mb_readwritegrd.c:29:
/usr/local/include/gmt/gmt_dev.h:164:10: fatal error: 'gmt_remote.h' file not found
#include "gmt_remote.h"          /* GMT type declarations */

It appears that the new header file gmt_remote.h is referenced in gmt_dev.h but is not actually installed with the other GMT header files in /usr/local/include/gmt/

I'm working on a Mac, with most GMT prerequisites installed with Homebrew.

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -659,7 +659,7 @@ if (BUILD_DEVELOPER)
 		gmt_dcw.h gmt_decorate.h gmt_defaults.h gmt_error.h gmt_error_codes.h gmt_fft.h gmt_gdalread.h gmt_grd.h
 		gmt_grdio.h gmt_hash.h gmt_io.h gmt_macros.h gmt_memory.h gmt_modern.h gmt_nan.h gmt_notposix.h gmt_plot.h
 		gmt_private.h gmt_project.h gmt_prototypes.h gmt_psl.h gmt_shore.h gmt_symbol.h gmt_synopsis.h
-		gmt_texture.h gmt_time.h gmt_types.h gmt_dev.h gmt_customio.h gmt_hidden.h gmt_mb.h
+		gmt_texture.h gmt_time.h gmt_types.h gmt_dev.h gmt_customio.h gmt_hidden.h gmt_mb.h gmt_remote.h
 		DESTINATION ${GMT_INCLUDEDIR}
 		COMPONENT Runtime)
 	install (FILES compat/qsort.h

--- a/src/gmt_dev.h
+++ b/src/gmt_dev.h
@@ -161,7 +161,7 @@ struct GMT_CTRL; /* forward declaration of GMT_CTRL */
 #include "gmt_plot.h"           /* extern functions defined in gmt_plot.c */
 #include "gmt_memory.h"         /* extern functions defined in gmt_M_memory.c */
 #include "gmt_types.h"          /* GMT type declarations */
-#include "gmt_remote.h"          /* GMT type declarations */
+#include "gmt_remote.h"         /* GMT remote dataset structure */
 
 #ifdef _OPENMP                  /* Using open MP parallelization */
 #include <omp.h>

